_vendors/processst.yaml

```diff
@@ -0,0 +1,8 @@
+---
+base_pricing: $100 per month
```

Comment:
This is a limited price tier for small startups only (staff and revenue limits) - can you update with the lowest normal business tier?

Reply:
Base price has now been updated